### PR TITLE
fix: show telemetry message extension source

### PR DESCRIPTION
**Pull request by brpowell**
### What does this PR do?

Fixes a small issue where the telemetry message prompt shows a null extension source.

### What issues does this PR fix or reference?

@W-8792554@

### Functionality Before

NullExtensionDescription

### Functionality After

Screen Shot 2021-01-26 at 10 01 10 AM

### CODE REVIEW

File: packages/salesforcedx-vscode-core/src/index.ts [hunk 1]
```diff
@@ -513,13 +515,40 @@ async function setupOrgBrowser(
...
+        if (selection && selection === showButtonText) {
+          vscode.commands.executeCommand(
+            'vscode.open',
+            vscode.Uri.parse(TELEMETRY_OPT_OUT_LINK)
+          );
+        }
+      });
+    context.globalState.update(TELEMETRY_GLOBAL_VALUE, true);
+  }
+}
```
Comment (brpowell) [inline packages/salesforcedx-vscode-core/src/index.ts:542]: Calling the showInformationMessage api in the utils package is what caused vscode to be confused about what extension was showing the message, so I moved that logic to the core extension. Lmk if you think this function belongs somewhere else in the src.

File: packages/salesforcedx-vscode-core/src/index.ts [hunk 2]
```diff
@@ -513,13 +515,40 @@ async function setupOrgBrowser(
...
+
 export async function activate(context: vscode.ExtensionContext) {
   const extensionHRStart = process.hrtime();
   const { name, aiKey, version } = require(context.asAbsolutePath(
     './package.json'
   ));
   await telemetryService.initializeService(context, name, aiKey, version);
-  telemetryService.showTelemetryMessage();
+  showTelemetryMessage(context);
+  // telemetryService.showTelemetryMessage();
```
Comment (brpowell) [inline packages/salesforcedx-vscode-core/src/index.ts:551]:
```suggestion
```

File: packages/salesforcedx-vscode-core/src/index.ts (hunk 1 above)
Comment (xyc) [reply packages/salesforcedx-vscode-core/src/index.ts:542]: Looks good - Just a reminder that this won't show a "null" extension in production, only in dev.

Comment (brpowell) [reply packages/salesforcedx-vscode-core/src/index.ts:542]: 😞

**Review by lcampos — APPROVED** [review]

**Pull request merged**